core/tests/puppeteer-acceptance-tests/utilities/user/logged-in-user.ts

```diff
@@ -79,6 +107,78 @@ export class LoggedInUser extends BaseUser {
     showMessage(`Subscribed to the creator with username ${username}.`);
   }
 
+  /**
+   * Navigates to the learner dashboard.
+   */
+  async navigateToLearnerDashboard(): Promise<void> {
+    await this.goto(learnerDashboardUrl);
+  }
+
+  /**
+   * Navigates to the progress section.
+   */
+  async navigateToProgressSection(): Promise<void> {
+    if (this.isViewportAtMobileWidth()) {
+      await this.clickOn(mobileProgressSectionSelector);
+    } else {
+      await this.page.waitForSelector(progressSectionSelector);
+      const progressSection = await this.page.$(progressSectionSelector);
+      if (!progressSection) {
+        throw new Error('Progress section not found.');
+      }
+      await progressSection.click();
+    }
+  }
+
+  /**
+   * Navigates to the community library tab of the learner dashboard.
+   */
+  async navigateToCommunityLessonsSection(): Promise<void> {
+    if (this.isViewportAtMobileWidth()) {
+      await this.clickOn(mobileProgressSectionSelector);
+      await this.clickOn('Lessons');
+    } else {
+      await this.page.click(communityLessonsSectionButton);
+    }
+  }
+
+  /**
+   * Navigates to the home section.
```

Comment:
I think it's better to mention it's home section from the learner dashboard menu. Ditto for the function below.

Reply:
Done.